labellines/line_label.py

```diff
@@ -162,6 +179,13 @@ def _update_anchors(self):
         x = self._line.convert_xunits(self._target_x)
         xdata, ydata = normalize_xydata(self._line)
 
+        # Convert timedelta to float if needed
```

Comment:
Note to self, can update when https://github.com/matplotlib/matplotlib/issues/8869 is resolved.